src/app/pages/CpsSty/index.jsx

```diff
@@ -118,10 +169,33 @@ const CpsStyContainer = ({ pageData }) => {
         aboutTags={aboutTags}
       />
       <ATIAnalytics data={pageData} />
-      <StyledGhostGrid as="main" role="main">
-        <Blocks blocks={blocks} componentsToRender={componentsToRender} />
-      </StyledGhostGrid>
-      <CpsRelatedContent content={relatedContent} />
+
+      <StyledGrid columns={gridColumns} enableGelGutters margins={gridMargins}>
+        <Grid
+          item
+          columns={gridColsMain}
+          startOffset={gridOffset}
+          as="main"
+          role="main"
+        >
+          <Blocks blocks={blocks} componentsToRender={componentsToRender} />
+          <CpsRelatedContent content={relatedContent} />
+        </Grid>
+        <GridSecondaryColumn item columns={gridColsSecondary}>
+          <ComponentWrapper>
+            <h2>This is a component in the second column</h2>
+          </ComponentWrapper>
+          <ComponentWrapper>
+            <h2>This is a component in the second column</h2>
+          </ComponentWrapper>
+          <ComponentWrapper>
+            <h2>This is a component in the second column</h2>
+          </ComponentWrapper>
+          <ComponentWrapper>
+            <h2>This is a component in the second column</h2>
+          </ComponentWrapper>
```

Comment:
Are we leaving these dummy components in for when we merge? I'd say we may as well as this page isn't going live without something _real_ to replace them.

Reply:
I think so, the only component we can use is `MostRead` and the ws-home team are working on a [PR](https://github.com/bbc/simorgh/pull/5437) to get that in Front Pages so when they've merged that it'll make it easy for us to put a real component in.

Reply:
Yeah, I figured we should probably leave them in to prevent the linter from complaining and to document how they're designed to be used.

Reply:
Have we given any more thought whether to create a component within Psammead for these? If not, maybe even just a Simorgh component?

Reply:
@12 That's a good question. If we ever made the layout reusable, we'd need to think about where `ComponentWrapper` would live. Until then, I think it makes sense to leave it where it is.